src/PrestaShopBundle/Install/DatabaseDump.php

```diff
@@ -66,6 +66,23 @@ private function __construct($dumpFile = null)
         $this->password = _DB_PASSWD_;
     }
 
+    private function getPdoDsn(): string
+    {
+        return 'mysql:host=' . $this->host . ';port=' . $this->port . ';dbname=' . $this->databaseName . ';charset=utf8';
```

Comment:
Duplicate from Db class, I'm sorry, but your PR brings no value here. I see no use case where it's needed to change the default behaviour

Reply:
If it can be deduped, please submit a code sudgestion.

Reply:
It's up to you to convince us to merge your PR, more if this PR brings no value :thinking:

Reply:
Pay me for that :)

I can stay confident that handling (and showing as well, which is not the case with exec) possible DB errors is enough value to merge this in.

Reply:
> Pay me for that :)

Would you pay me to review and merge your pull request?

Reply:
never and thanks for the PS t-shirt 8-)